lms/static/js/edxnotes/views/tabs/course_structure.js

```diff
@@ -4,10 +4,12 @@ define([
     'gettext', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
     'js/edxnotes/views/tab_view'
 ], function (gettext, _, NoteGroupView, TabPanelView, TabView) {
+    var view = "Location in Course";
     var CourseStructureView = TabView.extend({
+        view: 'Location in Course',
```

Comment:
I was wondering why we can't use `view` on line-9 which is `view: 'Location in Course',` just like we did at line 12 i.e. using the view variable instead of hardcoded string.